classes/PaymentModule.php

```diff
@@ -1108,7 +1108,7 @@ protected function createOrderCartRules(
             // THEN
             //  The voucher is cloned with a new value corresponding to the remainder
             $remainingValue = $cartRule->reduction_amount - $values[$cartRule->reduction_tax ? 'tax_incl' : 'tax_excl'];
-            $remainingValue = Tools::ps_round($remainingValue, $computingPrecision);
+            $remainingValue = Tools::ps_round($remainingValue, _PS_PRICE_COMPUTE_PRECISION_);
```

Comment:
Nope not in 177 where computing precision depends on currency precision

Reply:
But it's a new change in the 1.7.6.x :thinking: